ui/src/pages/ConcurrencyLimit.vue

```diff
@@ -36,7 +36,8 @@
     { label: 'Details', hidden: media.xl },
     { label: 'Active Task Runs' },
   ])
-  const { tabs } = useTabs(computedTabs)
+  const tab = useRouteQueryParam('tab', 'Details')
```

Comment:
What's the behavior like if the details tab is hidden on smaller screens at the outset?

Reply:
That's what I was hoping - nice composition